app/web/features/connections/friends/useFriendList.test.tsx

```diff
@@ -43,18 +45,20 @@ describe("when the listFriends query succeeds", () => {
     await waitForNextUpdate();
 
     // Called twice since the user has two friends in the fixture data
-    expect(getUserMock).toHaveBeenCalledTimes(2);
+    expect(getLiteUsersMock).toHaveBeenCalledTimes(1);
     expect(result.current).toEqual({
       data: [users[1], users[2]],
+      friendIds: [users[1].userId, users[2].userId],
       errors: [],
       isError: false,
       isLoading: false,
     });
   });
 
-  it("returns isLoading as true with no errors if some getUser queries are loading", async () => {
-    getUserMock.mockImplementation((userId: string) => {
-      return userId === "2" ? getUser(userId) : new Promise(() => void 0);
+  it("returns isLoading as true with no errors if some getLiteUsers query is loading", async () => {
+    // eslint-disable-next-line unused-imports/no-unused-vars
+    getLiteUsersMock.mockImplementation((userIds: number[]) => {
+      return new Promise(() => void 0);
     });
```

Comment:
Remove this eslint disable comment, and just take out the function argument `userIds` since you're not using it. That will fix it!